component/azstorage/azstorage.go

```diff
@@ -573,7 +573,7 @@ func NewazstorageComponent() internal.Component {
 		stConfig: AzStorageConfig{
 			blockSize:      0,
 			maxConcurrency: 32,
-			defaultTier:    getAccessTierType("none"),
+			defaultTier:    getAccessTierType(""),
```

Comment:
should it be 'nil' or empty string here?

Reply:
`getAccessTierType()` method returns nil when empty string is passed.